src/views/blocks/rsvp/form/error.php

```diff
@@ -17,6 +17,6 @@
 ?>
 <div class="tribe-block__rsvp__message__error">
 
-	<?php esc_html_e( 'Please fill in the RSVP confirmation name and email fields.', 'event-tickets' ); ?>
+	<?php printf( esc_html__( 'Please fill in the %s confirmation name and email fields.', 'event-tickets' ), tribe_get_rsvp_label_singular( basename( __FILE__ ) ) ); ?>
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: b2d42bdd8abb0a2a160e52c4f77619ff</sub>